changelog/62483.fixed

```diff
@@ -0,0 +1 @@
+Update sanitizing masking for Salt SSH to include additional password like strings.
```

Comment:
I just noticed this changelog has a different number as https://github.com/saltstack/salt/pull/62482/files this would mean in the 3006 release we would be adding this as fixed even though it was fixed in 3005. We can either rename this filename to 62482 and the merge forward will capture that the file was deleted. or just remove this changelog.

Reply:
woops i meant for this to be on the review of the master branch PR. will post there